pkg/detectors/brandfetch/v2/brandfetch_test.go

```diff
@@ -0,0 +1,120 @@
+//go:build detectors
+// +build detectors
+
+package brandfetch
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/kylelemons/godebug/pretty"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+func TestBrandfetch_FromChunk(t *testing.T) {
```

Comment:
These test cases for v2 might fail as the regex is different and these v2 secrets does not exist in the vault.
I am not sure what to do here so tagging experts 👨🏻‍💻 @zricethezav @mcastorina @abmussani 

And please add the the pattern test cases for both versions separately.

Reply:
<img width="723" alt="Screenshot 2024-10-28 at 8 46 53 PM" src="https://github.com/user-attachments/assets/7071fb2d-57e7-4964-baf6-50d44d8fa26d">

Didn't get your point here, tests are passing, do you mean to add the tests which will fail in v2 but are of v1 pattern?

Reply:
No worries! The pattern test cases will be covered as part of https://github.com/trufflesecurity/trufflehog/pull/3559